src/harness/harness.ts

```diff
@@ -896,7 +896,8 @@ namespace Harness {
             { name: "fileName", type: "string" },
             { name: "libFiles", type: "string" },
             { name: "noErrorTruncation", type: "boolean" },
-            { name: "suppressOutputPathCheck", type: "boolean" }
+            { name: "suppressOutputPathCheck", type: "boolean" },
+            { name: "moduleResolution", type: "string"}

```

Comment:
do we need this, I think that `moduleResolution` will already work since it is legal command line option